uber/custom_tags.py

```diff
@@ -445,10 +445,12 @@ def render(self, context):
 class single_day_prices(template.Node):
     def render(self, context):
         prices = ''
+        skiprest = ''
```

Comment:
What a memory!  This fixed a bug where extra crap was displayed on the "price goes up" line, IIRC.  It also looks like V1 of the fix.  I think V2 used "break" per your suggestion.  Let me see what I can find.